tap_clickup/schemas/space.json

```diff
@@ -1,81 +1,108 @@
 {
     "definitions": {
-        "user": {
-            "type": "object",
-            "properties": {
-                    "id": {
-                        "type": "integer"
-                    },
-                    "username": {
-                        "type": "string"
-                    },
-                    "email": {
-                        "type": ["null", "string"]
-                    },
-                    "color": {
-                        "type": ["null", "string"]
-                    },
-                    "profilePicture": {
-                        "type": ["null", "string"]
-                    },
-                    "initials": {
-                        "type": ["null", "string"]
-                    },
-                    "role": {
-                        "type": ["null", "integer"]
-                    },
-                    "custom_role": {
-                        "type": ["null", "integer"]
-                    },
-                    "last_active": {
-                        "type": ["null", "string"]
-                    },
-                    "date_joined": {
-                        "type": ["null", "string"]
-                    },
-                    "date_invited": {
-                        "type": ["null", "string"]
-                    }
-            }
-        },
         "members": {
-            "type": ["array", "null"],
-            "items": [
-                {
+            "type": ["null", "object"],
+            "properties": {
+                "user": {
                     "type": "object",
                     "properties": {
-                        "user": {
-                            "$ref": "#/definitions/user"
```

Comment:
Would be nice to keep this instead of duplicating what a user is

Reply:
Nested definitions were giving me some issues, posted https://github.com/transferwise/pipelinewise-singer-python/issues/55 instead . Created a tracking issue on our end here https://github.com/AutoIDM/tap-clickup/issues/109